### ISSUE TITLE
`cloudknot clean` cli

### ISSUE THREAD

**Issue by arokem** [richford/cloudknot#136]
If you run this enough times, and are not sufficiently diligent about clobbering resources from failed attempts, you might end up with a rather large cloudknot configuration file (and a lot of dangling resources on AWS?). Might be nice to have a CLI that allows you to clean up everything, after all the knots are no longer existing.

**Comment by arokem** [comment]
It just occurred to me that you can 'revive' dangling knots by referring to them by name, so maybe this is not neccessary.

**Comment by richford** [comment]
Could possibly be merged with #8. If possible, I'd like to hold off on this and #8 until after #125. I think the way we use the config file will change dramatically after switching over to cloud formation.

Having said that, I don't think I'll get to #125 for a while. I think it's low priority for me with the other stuff on my plate. Let me know if you think differently and I can redirect.

**Comment by arokem** [comment]
No hurry on this. I think that reviving these dangling resources by reference to their name covers this for now. We can revisit once #125 is addressed, but again -- no hurry!

**Comment by richford** [comment]
Once we merge #250, I think we could write the CLI for this. The CLI should prune the config file, but also have an interactive part to remove resources on AWS. Once we prune the config file to refer only to existing resources, we can then iterate through the config resources and ask the user if they want to keep them on AWS using `[y, n, q]` input on the command line.

**Comment by arokem** [comment]
Yes - a CLI that asks you whether or not to nuke unused resources without having to go into the console would be awesome.
